vignettes/cookbook_advanced_use_cases.Rmd

```diff
@@ -14,10 +14,520 @@ knitr::opts_chunk$set(
+
+**Example**
+
+Assuming the user operates under regulation that matches the scope of PACTA at least for parts of the reporting, the following could be an imaginable scenario of reporting. For a materiality assessment of loan book exposures to climate relevant sectors, users may want to map the loans in their loan book to PACTA sectors. For example, for a bank that lends to clients in PACTA scope in power generation and steel manufacturing, and  has some additional clients that are not in scope of any of the PACTA sectors, the materiality assessment could be based on the following calculation:
+
+```{r use_case_one_materiality_calculation}
+materiality <- coverage %>% 
+  mutate(sector = if_else(!sector %in% c("power", "cement"), "not in scope", sector)) %>% # we assume this bank only lends to cement, power, and not-in-scope companies
+  summarise(
+    sector_sum_loan_size_outstanding = sum(loan_size_outstanding),
+    .by = sector
+  ) %>%
+  mutate(
+    sector_share_loan_size_outstanding = round(sector_sum_loan_size_outstanding / sum(sector_sum_loan_size_outstanding), 2)
+  )
+```
+
+```{r use_case_one_materiality_result, echo = FALSE}
+result_table(materiality, title = "Materiality assessment", subtitle = "loan book exposures to climate relevant sectors")
+```
+
+The bank might report to the supervisor along the lines of:
+-	`r pull(filter(materiality, sector == "power"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r pull(filter(materiality, sector == "power"), sector_sum_loan_size_outstanding)` USD) go towards power generation
+-	`r pull(filter(materiality, sector == "cement"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r pull(filter(materiality, sector == "cement"), sector_sum_loan_size_outstanding)` USD) go towards cement production
+-	`r pull(filter(materiality, sector == "not in scope"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r pull(filter(materiality, sector == "not in scope"), sector_sum_loan_size_outstanding)` USD) go towards other sectors that are not climate relevant
+
+This can be a direct outcome based on the matched and prioritized data set compared with values in the raw loan book data set. (((Note that if sectors in adjacent parts of the value chain are required too, it may be an idea to include loans with both flags: borderline TRUE and FALSE.)))
```

Comment:
is this supposed to be wrapped in `(((` `)))`?
```suggestion
This can be a direct outcome based on the matched and prioritized data set compared with values in the raw loan book data set. (Note that if sectors in adjacent parts of the value chain are required too, it may be an idea to include loans with both flags: borderline TRUE and FALSE.)
```

Reply:
tbh, I am a bit uncertain about this section/comment. It feels like explaining this properly would take significantly more space. I put it in ((())) to note to myself that this should either be explained more or removed.

Reply:
moved this whole paragraph to "Notes and caveats" section